search/cypress/integration/weedai_upload.js

```diff
@@ -61,6 +65,63 @@ describe('overall upload workflow', () => {
         cy.clickText(/^Submit$/)
     })
 
+    it('Test Cvat Upload', function() {
+        const label = 'weed: UNSPECIFIED'
+
+        const task = {
+            name: `task_${Math.random().toString(36).slice(-4)}`,
+            label: label,
+            image: 'test_cvat/003_image.jpg',
+            createRectangleShape2Points: {
+                points: 'By 2 Points',
+                type: 'Shape',
+                labelName: label,
+                firstX: 250,
+                firstY: 350,
+                secondX: 350,
+                secondY: 450,
+            }
+        }
+
+        cy.visit('http://localhost/cvat-annotation/auth/register', {failOnStatusCode: false})
+        cy.wait(20000)
```

Comment:
is there not a way in cypress to wait until something appears on screen, with a maximum timeout, rather than waiting for a fixed (and large) amount.
20s seems to be a lot!